src/schema/project-scratch-def/features.ts

```diff
@@ -25,5 +25,3 @@ export const FeaturesSchema = z.array(
     z.enum(simpleFeaturesList),
   ])
 );
-
-export type Features = z.infer<typeof FeaturesSchema>;
```

Comment:
some unused exports in the previous pr.  this pr adds the `knip` which we've found really useful at reducing ai slop